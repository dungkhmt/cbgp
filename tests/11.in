30 30
30 43
17 12
12 22
22 24
24 9
9 4
4 23
23 20
20 17
2 28
28 14
14 8
8 15
15 3
3 19
19 5
5 30
30 2
27 13
13 25
25 6
6 27
11 16
16 26
26 7
7 18
18 11
10 21
21 29
29 1
1 10
28 12
27 2
8 4
7 1
19 20
22 9
30 17
10 11
25 15
14 13
5 6
23 3
29 16
1
15 3 21 11 7 1 0 0 30 2 12 2 27 3 7 3 16 16 17 5 19 7 19 15 12 4 10 6 8 2 23 4 30 4 21 9 31 1 34 0 19 5 18 16 6 0 17 17 10 2 25 4 13 3 18 14 20 4 29 3 
