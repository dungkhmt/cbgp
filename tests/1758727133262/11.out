(16, 11), (11, 21), (21, 23), (23, 8), (8, 3), (3, 22), (22, 19), (19, 16), (1, 27), (27, 13), (13, 7), (7, 14), (14, 2), (2, 18), (18, 4), (4, 29), (29, 1), (26, 12), (12, 24), (24, 5), (5, 26), (10, 15), (15, 25), (25, 6), (6, 17), (17, 10), (9, 20), (20, 28), (28, 0), (0, 9), (27, 11), (26, 1), (7, 3), (6, 0), (18, 19), (21, 8), (29, 16), (9, 10), (24, 14), (13, 12), (4, 5), (22, 2), (28, 15), 
numberIntersection=0 
minDistance=3.0951642250519726 
minAngle=0.5112307122777899 
sumAngle=43.393378994693244 
sumDistance=45820.00796587858 
minEdgeLength=3.605551275463989 
sumEdgeLength=736.9349144784101 

0.0000000000 3.0951642251 0.5112307123 43.3933789947 45820.0079658786 3.6055512755 736.9349144784

0: (33, 11),
1: (41, 42),
2: (15, 4),
3: (1, 7),
4: (61, 7),
5: (30, 8),
6: (54, 12),
7: (11, 12),
8: (1, 116),
9: (34, 21),
10: (40, 33),
11: (35, 56),
12: (22, 14),
13: (14, 18),
14: (15, 8),
15: (42, 18),
16: (64, 16),
17: (44, 28),
18: (58, 4),
19: (65, 0),
20: (37, 19),
21: (4, 120),
22: (14, 0),
23: (0, 120),
24: (21, 10),
25: (48, 15),
26: (27, 14),
27: (29, 52),
28: (39, 16),
29: (59, 14),

Time taken: 7.777 s
33 11 41 42 15 4 1 7 61 7 30 8 54 12 11 12 1 116 34 21 40 33 35 56 22 14 14 18 15 8 42 18 64 16 44 28 58 4 65 0 37 19 4 120 14 0 0 120 21 10 48 15 27 14 29 52 39 16 59 14 
0: (33, 11), 1: (41, 42), 2: (15, 4), 3: (1, 7), 4: (61, 7), 5: (30, 8), 6: (54, 12), 7: (11, 12), 8: (1, 116), 9: (34, 21), 10: (40, 33), 11: (35, 56), 12: (22, 14), 13: (14, 18), 14: (15, 8), 15: (42, 18), 16: (64, 16), 17: (44, 28), 18: (58, 4), 19: (65, 0), 20: (37, 19), 21: (4, 120), 22: (14, 0), 23: (0, 120), 24: (21, 10), 25: (48, 15), 26: (27, 14), 27: (29, 52), 28: (39, 16), 29: (59, 14), 
numberIntersection=0 minAngle=0.5112307122777899 sumAngle=43.39337899471073 minEdgeLength=3.605551275463989 sumEdgeLength=736.9349144784102 minDistance=3.0951642250519726 sumDistance=45820.007965879224 
